src/frontend/context/ObservationsContext.js

```diff
@@ -31,8 +31,19 @@ export type Observation = {
   userId?: string,
   type: "observation",
   links?: string[],
-  schemaVersion: 4,
+  schemaVersion: number,
```

Comment:
This was actually intended to be `4`, rather than `number`, since when this changes the type is different.

Reply:
ah makes sense. will have to make some adjustments elsewhere if we restore this

Reply:
just realized the `convertFromServer` function I highlighted in above comment is meant for converting observations from `3` to `4`, so can disregard

Reply:
actually scratch that – i'm a little bit confused about what's going on. I'm assuming that the type from `mapeo-schema` should reflect what's being returned by the server right? however, looks like we're inserting/updating observation values we persist as version 3:

```ts
    createObservation: async (
      value: ObservationValue
    ): Promise<Observation> => {
      const valueForServer = {
        ...value,
        type: "observation",
        schemaVersion: 3,
      };
      const serverObservation = (await post(
        "observations",
        valueForServer
      )) as ServerObservation;
      return convertFromServer(serverObservation);
    },
```

Based on https://github.com/digidem/mapeo-schema/blob/master/docs/observation.md#schemaversion, the server version _should_ be 4 but that doesn't align with what's currently implemented. Is our implementation just outdated or am I missing something?

Reply:
_ermmm..._ I am also confused, this does not seem correct since I thought we were using schema version `4`. I might have got in a muddle about this, or forgotten to update this piece.